tests/top_bar_component.test.ts

```diff
@@ -975,10 +975,10 @@ test("Clicking on a topbar button triggers three renders", async () => {
 
   await click(fixture, ".o-spreadsheet-topbar [title='Bold (Ctrl+B)']");
 
-  // two renders from the collaboration session
+  // one renders from the collaboration session
```

Comment:
the comment does not match the actual situation (while I agree it's a nightmare here)

Reply:
Indeed, I fixed the comment :)